src/pages/code-for-good.js

```diff
@@ -0,0 +1,317 @@
+import React from 'react';
+import Layout from '../components/dev-hub/layout';
+import { Helmet } from 'react-helmet';
+import { useSiteMetadata } from '../hooks/use-site-metadata';
+import {
+    colorMap,
+    fontSize,
+    screenSize,
+    size,
+    lineHeight,
+} from '../components/dev-hub/theme';
+import { H3, ArticleH2, ArticleH4, P } from '../components/dev-hub/text';
+import styled from '@emotion/styled';
+import Button from '../components/dev-hub/button';
+import HeroBanner from '../components/dev-hub/hero-banner';
+import BackgroundWaves from '../images/code-for-good/background-waves.png';
+import WildAidLogo from '../images/code-for-good/wild-aid-logo.png';
+import WildAidDashboard from '../images/code-for-good/wild-aid-dashboard.png';
+import VesselRecord from '../images/code-for-good/vessel-record.png';
+import Link from '../components/dev-hub/link';
+
+const CAPTION_MAX_WIDTH = '345px';
+const BODY_TEXT_MAX_WIDTH = '320px';
+const HERO_TEXT_WIDTH = '425px';
+const DASHBOARD_IMAGE_WIDTH = 780;
+const VESSEL_IMAGE_WIDTH = 345;
+const LOGO_IMAGE_WIDTH = 350;
+const LOGO_IMAGE_HEIGHT = 450;
+
+const Header = styled('header')`
+    display: flex;
+    justify-content: space-between;
+    padding-top: ${size.large};
+    @media ${screenSize.upToLarge} {
+        flex-direction: column-reverse;
+        margin-bottom: ${size.large};
+    }
+`;
+const HeaderText = styled('div')`
+    margin-bottom: -${size.xlarge};
+    max-width: ${HERO_TEXT_WIDTH};
+`;
+
+const HeaderLink = styled(Link)`
+    color: ${colorMap.darkGreen};
+    display: inline;
+    margin-top: ${size.large};
+    margin-left: ${size.mediumLarge};
+    white-space: nowrap;
+    @media ${screenSize.upToLarge} {
+        display: block;
+        margin: 0;
+    }
+`;
+
+const StyledHeroBanner = styled(HeroBanner)`
+    margin: 0 ${size.medium};
+    @media ${screenSize.upToLarge} {
+        margin: 0;
+    }
+`;
+
+const StyledP = styled(P)`
+    margin-top: ${size.articleContent};
+    margin-bottom: ${size.xlarge};
+`;
+
+const BodyText = styled(P)`
+    margin-top: ${fontSize.default};
+`;
+
+const StyledButton = styled(Button)`
+    margin-top: ${size.large};
+    @media ${screenSize.upToLarge} {
+        margin-bottom: ${size.medium};
+    }
+`;
+
+const BodyContent = styled('div')`
+    margin: 0 auto;
+    max-width: ${size.maxContentWidth};
+    width: 100%;
+    @media ${screenSize.upToMedium} {
+        padding: 0 ${size.default};
+    }
+`;
+
+const StyledParagraphLink = styled('a')`
```

Comment:
Had to use `<a/>` here instead of `Link` because the text color wasn't showing up for some reason (on both chrome and firefox)

Reply:
Hmm this should definitely be a `Link`. Check out the `Link` styles, is there somewhere we can override color here? We should be able to achieve that with CSS